From 5df6a4506d060fcc3b875437cea8d77b5950325d Mon Sep 17 00:00:00 2001
From: Cheng-Yang Chou <yphbchou0911@gmail.com>
Date: Wed, 1 Jul 2026 01:05:35 +0800
Subject: [PATCH 01519/15333] sched_ext: Parenthesize local SCX_EVENT defines
 to fix checkpatch error

Fix checkpatch "Macros with complex values should be enclosed in
parentheses" error.

Suggested-by: Andrea Righi <arighi@nvidia.com>
Signed-off-by: Cheng-Yang Chou <yphbchou0911@gmail.com>
Signed-off-by: Tejun Heo <tj@kernel.org>
---
 kernel/sched/ext/ext.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/sched/ext/ext.c b/kernel/sched/ext/ext.c
index 0c51f9826a77..a9d23d9f5947 100644
--- a/kernel/sched/ext/ext.c
+++ b/kernel/sched/ext/ext.c
@@ -4979,7 +4979,7 @@ static ssize_t scx_attr_events_show(struct kobject *kobj,
 	int at = 0;
 
 	scx_read_events(sch, &events);
-#define SCX_EVENT(name)	at += scx_attr_event_show(buf, at, &events, name)
+#define SCX_EVENT(name)	(at += scx_attr_event_show(buf, at, &events, name))
 	SCX_EVENTS_LIST(SCX_EVENT);
 #undef SCX_EVENT
 	return at;
@@ -10309,7 +10309,7 @@ static void scx_read_events(struct scx_sched *sch, struct scx_event_stats *event
 	memset(events, 0, sizeof(*events));
 	for_each_possible_cpu(cpu) {
 		struct scx_event_stats *e_cpu = &per_cpu_ptr(sch->pcpu, cpu)->event_stats;
-#define SCX_EVENT(name)	events->name += READ_ONCE(e_cpu->name)
+#define SCX_EVENT(name)	(events->name += READ_ONCE(e_cpu->name))
 		SCX_EVENTS_LIST(SCX_EVENT);
 #undef SCX_EVENT
 	}
-- 
2.55.0

